frontend/src/Metamaps/Router.js

```diff
@@ -202,6 +202,11 @@ const _Router = Backbone.Router.extend({
 
 const Router = new _Router()
 
+Router.navigateAndTrack = (fragment, options) => {
+  Router.navigate(fragment, options)
+  ga && ga('send', 'pageview')
```

Comment:
Can you change this to `window.ga && window.ga('send', 'pageview')`? Stuff like this always causes trouble if you're in a node env (e.g. testing)